.github/workflows/upload-sbom.yaml

```diff
@@ -0,0 +1,24 @@
+name: Syft SBOM Creation
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  upload-sbom:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      security-events: write
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Docker image
+        run: make build
```

Comment:
We should provide a custom `DOCKER_TAG` and then use in the image name below.

https://github.com/mirego/killswitch/blob/main/Makefile#LL4C5-L4C5